Lib/ldap/schema/subentry.py

```diff
@@ -474,8 +474,11 @@ def urlfetch(uri,trace_level=0,bytes_mode=None):
     l.unbind_s()
     del l
   else:
-    import urllib,ldif
-    ldif_file = urllib.urlopen(uri)
+    import ldif
+    from ldap.compat import urlopen
+    if not uri.lower().startswith("file://"):
```

Comment:
It worked with python2, so backward compatibility. And according the pyldap code, the code itself expects just filename, I'm not sure if we should issue a warning